xikolo-tvos/assets-tvos.xcassets/item-lti_exercise-160.imageset/item-lti_exercise-160.svg

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Are you sure svg files work? I used to have to convert all vector graphics to pdf.